cmd/task-runner/task-runner.go

```diff
@@ -156,6 +159,18 @@ func Run(build BuildFlags) {
 	runner := task.NewRunner(cli.runnerOpts)
 	ctx := contextUntilSignal(context.Background(), syscall.SIGINT, syscall.SIGTERM)
 	eg, ctx := errgroup.WithContext(ctx)
+
+	if cli.runnerOpts.VodDecryptPrivateKey != "" && cli.runnerOpts.VodDecryptPublicKey != "" {
```

Comment:
Can you add a // TODO everywhere you can saying that this should be moved to Catalyst-only? It hurts that we need this hack, but I agree with the approach not to need a huge refactor.